cluster/charts/crossplane/values.yaml.tmpl

```diff
@@ -15,14 +15,14 @@ imagePullSecrets:
 
 clusterStacks:
   aws:
-    version: v0.1.0
+    version: v0.2.0
```

Comment:
Do you think it would be a good idea to have these pointed at `master` versions on `master`, then point to latest on `release-0.4`?

Reply:
For master:
Makes sense to me but this was actually suggested by @prasek https://github.com/crossplaneio/crossplane/pull/932#pullrequestreview-301925590

For `release-0.4`
I didn't know `latest`, are they maintained by our release process and point to the versions that I updated here? If so, this would definitely be better.

Reply:
It looks like @prasek just meant that Crossplane releases should be pinned to stable stacks, which I definitely agree with! I think any time we cut a release we should pin to most recent stable version of each stack, but in master, we should point to stack master versions (the `master` tag is equivalent to `latest` for us). Therefore, in this PR I think it would be useful to pin to `master` versions and then after cherry-picking onto `release-0.4` we should update the versions to stable.

Reply:
Agreed, done!
My only concern is, we should not forget setting to stable versions before making a release. But this is already valid and looks like it is already forgotten on `release-0.4` :)

Reply:
No, I am wrong, `release-0.4` already points to correct thanks to @jbw976